dev/io.openliberty.data.internal.persistence/src/io/openliberty/data/internal/persistence/service/DBStoreEMBuilder.java

```diff
@@ -814,13 +825,13 @@ public void generate(Writer out) throws Exception {
         // Note that exceptions thrown here or by the persistence service will be logged by the
         // direct caller (FutureEMBuilder) or the DDL generation MBean.
         if (persistenceServiceUnit == null) {
-            throw new IllegalStateException("EntityManagerFactory for Jakarta Data repository has not been initialized for the " + databaseStoreId + " DatabaseStore.");
+            throw new IllegalStateException("EntityManagerFactory for Jakarta Data repository has not been initialized for the " + configDisplayId + " DatabaseStore.");
         }
         persistenceServiceUnit.generateDDL(out);
     }
 
     @Override
     public String getDDLFileName() {
-        return databaseStoreId + "_repository";
+        return configDisplayId + "_jakartaData";
```

Comment:
Would this look better to the user with a capital letter for the spec name?
It would also help make the distinction between a DDL file for a product function vs a technology/specification for the user.
```suggestion
        return configDisplayId + "_JakartaData";
```

Reply:
Yeah, I think that does look better especially since Jakarta is a proper noun (organization name).